addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10416,7 +10425,7 @@ msgid "Scanning music videos using %s"
 msgstr ""
 
 msgctxt "#20319"
-msgid "Scanning tvshows using %s"
+msgid "Scanning TV shows using %s"
```

Comment:
+10 for this change, this has annoyed me since so far back I can't even remember!